modules/@apostrophecms/piece-type/ui/apos/components/AposPiecesManager.vue

```diff
@@ -124,6 +132,20 @@ export default {
     moduleName: {
       type: String,
       required: true
+    },
+    join: {
+      type: Boolean,
+      default: false
+    },
+    items: {
```

Comment:
This needs to be named more specifically. We already have a `pieces` property that's populated in `getPieces`. If this property is specific to the relationship context then let's be clear about that (especially to help with refactoring).

Reply:
`pieces` are only the 10 pieces displayed in the current "page" of pieces in the modal. `items` is a prop, becoming the data `selectedItems` (`selectedItems` are the ones used throughout the component) and holds the list of selected pieces in all "pages" of pieces. I don't think it will be only useful in a `relationship` context (although it is at the moment). Do you prefer `initiallySelectedItems`?

Reply:
Yes, `initiallySelectedItems` sounds good, thanks. That will help if it needs to be part of the refactor.

Reply:
Done